services/app-api/testHelpers/emailerHelpers.ts

```diff
@@ -0,0 +1,212 @@
+    import {EmailConfiguration, EmailData,
+        Emailer,
+        newPackageCMSEmailTemplate,
+        newPackageStateEmailTemplate,
+    } from '../emailer'
+    import { StateSubmissionType, CognitoUserType, CognitoStateUserType } from '../../app-web/src/common-code/domain-models'
+
+       
+    const testEmailConfig: EmailConfiguration = {
+        stage: 'LOCAL',
+        baseUrl: 'http://localhost',
+        emailSource: 'emailSource@example.com',
+        cmsReviewSharedEmails: ['cmsreview1@example.com', 'cmsreview2@example.com']
+   }
+    const testEmailer = (customConfig?: EmailConfiguration): Emailer => {
+        const config = customConfig || testEmailConfig
+        return {
+            sendEmail: jest.fn(
+                async (emailData: EmailData): Promise<void | Error> => {
+                    console.log('Email content' + emailData)
+                }
+            ),
+            sendCMSNewPackage: function async(
+                submission: StateSubmissionType
+            ): Promise<void | Error> {
+                const emailData = newPackageCMSEmailTemplate(submission, config)
+                return this.sendEmail(emailData)
+            },
+            sendStateNewPackage: function async(
+                submission: StateSubmissionType,
+                user: CognitoUserType
+            ): Promise<void | Error> {
+                const emailData = newPackageStateEmailTemplate(
+                    submission,
+                    user,
+                    config
+                )
+                return this.sendEmail(emailData)
+            },
+        }
+    }
+    
+   const mockUser = (): CognitoStateUserType  => {
+        return {
+            role: 'STATE_USER',
+            email: 'test+state+user@example.com',
+            name: 'Test State User',
+            state_code: 'MN',
+            }
+   }
+   
+   const mockContractAndRatesSubmission = (
```

Comment:
I bet at some point we'll want to break out these mock submissions somewhere else, but I think it makes sense for them to be here for now since they are only used in email tests rn.